File: docs/docs/API-Reference/python-examples/api-logs/retrieve-logs-with-optional-parameters.py
import os

import requests

url = f"{os.getenv('LANGFLOW_URL', '')}/logs?lines_before=0&lines_after=0&timestamp=0"

headers = {
    "accept": "application/json",
    "x-api-key": f"{os.getenv('LANGFLOW_API_KEY', '')}",
}

response = requests.request("GET", url, headers=headers)
response.raise_for_status()

print(response.text)

File: docs/versioned_docs/version-1.12.0/API-Reference/javascript-examples/api-logs/retrieve-logs-with-optional-parameters.js
const url = `${process.env.LANGFLOW_URL ?? ""}/logs?lines_before=0&lines_after=0&timestamp=0`;

const options = {
  method: 'GET',
  headers: {
    "accept": `application/json`,
    "x-api-key": `${process.env.LANGFLOW_API_KEY ?? ""}`,
  },
};

fetch(url, options)
  .then(async (response) => {
    if (!response.ok) {
      throw new Error(`HTTP ${response.status}`);
    }
    const text = await response.text();
    console.log(text);
  })
  .catch((error) => console.error(error));
